README.md

```diff
@@ -1 +1,38 @@
-# eslint-plugin-yola
\ No newline at end of file
+# eslint-plugin-yola
+
+Extendable eslint plugin that ensures code style quality for Yola front-end codebase.  
+ES6 code style is based on [Airbnb Javascript Style Guide](https://github.com/airbnb/javascript).
+
+This plugin solves problem of shareable eslint configs that depend on eslint plugins, which described in https://github.com/eslint/eslint/issues/3458.  
+Basically, to resolve eslint plugins dependencies this plugin wraps `eslint-config-airbnb`, `eslint-config-airbnb-base` rules with extracted rules from plugins. The approach was inspired by [eslint-plugin-react-app](https://github.com/mmazzarolo/eslint-plugin-react-app/) 
+
+# Installation
```

Comment:
Treat your markdown like any other document. Make sure the headers have hierarchy.

* generally there is one h1 (the title of the repo)
* these should all be h2's `##`